modules/auth/auth.service.js

```diff
@@ -17,12 +18,18 @@ const login = async (body) => {
   const existedUser = await userService.findByUsername(body.username);
 
   if (!existedUser) {
-    throw new Error('Username is not existed!');
+    throw new APIError({
+      message: 'Username is not existed!',
+      status: 400
```

Comment:
should it be `404`?

`400` Bad Request: The server could not understand the request due to invalid syntax.
`404` Not Found:  The server can not find the requested resource. In the browser, this means the URL is not recognized. **In an API, this can also mean that the endpoint is valid but the resource itself does not exist**. Servers may also send this response instead of 403 to hide the existence of a resource from an unauthorized client. This response code is probably the most famous one due to its frequent occurrence on the web.

In my opinion, it should be 404 because of non-existed resources, not invalid syntax

Reply:
Yeah that my mistake